tasks/db_deploy/db_deploy.py

```diff
@@ -6,376 +6,424 @@
+    Args:
+        con: A connection to the database.
 
-        Returns:
-            string: description of status of create schema.
+    Returns:
+        Description of status of create schema.
 
-        Raises:
-            DatabaseError: An error occurred.
+    Raises:
+        DatabaseError: An error occurred.
     """
-    platform = os.environ["PLATFORM"]
+    platform = os.environ[OS_ENVIRON_PLATFORM_KEY]
     _LOG.info(f"platform: {platform}")
     cur = get_cursor(con)
     if platform == "AWS":
         sql_stmt = """SET SESSION AUTHORIZATION dbo;"""
-        status = execute_sql(cur, sql_stmt, "auth dbo")
+        execute_sql(cur, sql_stmt, "auth dbo")
 
     sql_file = f"schema{SEP}app.sql"
     status = execute_sql_from_file(cur, sql_file, "create schema")
     cur.close()
     con.commit()
     return status
 
-def create_tables():
+
+def create_tables(db_host: str, db_name: str, db_port: str, db_user: str, db_password: str) -> None:
     """
     Creates the database tables.
 
-        Args:
-            con (object): a connection to the new database just created
+    Args:
+        db_host: The database host.
+        db_name: The name of the database.
+        db_port: The database port.
+        db_user: The username to connect to the database with.
+        db_password: The password to connect to the database with.
 
-        Returns:
-            string: description of status of create tables.
-
-        Raises:
-            DatabaseError: An error occurred.
+    Raises:
+        DatabaseError: An error occurred.
     """
-    ddl_dir = os.environ["DDL_DIR"]
+    ddl_dir = os.environ[OS_ENVIRON_DDL_DIR_KEY]  # todo: Move close to other os.environ
     table_dir = f"{ddl_dir}tables"
-    sql_files = get_files_in_dir(table_dir)
-    for file in sql_files:
+    sql_file_names = get_file_names_in_dir(table_dir)
+    for file in sql_file_names:
         sql_file = f"tables{SEP}{file}"
+        # todo: Does this need to be repeated?
+        con = get_db_connection(db_host, db_name, db_port, db_user, db_password)
+        cur = get_cursor(con)  # todo: Does this need to be repeated?
+        sql_stmt = """SET SESSION AUTHORIZATION dbo;"""  # todo: Does this need to be repeated?
+        execute_sql(cur, sql_stmt, "auth dbo")
         try:
-            con = get_db_connnection()
-            cur = get_cursor(con)
-            sql_stmt = """SET SESSION AUTHORIZATION dbo;"""
-            status = execute_sql(cur, sql_stmt, "auth dbo")
-            status = execute_sql_from_file(cur, sql_file, f"create table in {sql_file}")
-            con.close()
+            execute_sql_from_file(cur, sql_file, f"create table in {sql_file}")
         except ResourceExists as dd_err:
             _LOG.warning(f"ResourceExists: {str(dd_err)}")
-            status = log_status(f"table in {sql_file} already exists")
-    return status
+            log_status(f"table in {sql_file} already exists")
```

Comment:
Consider changing to _LOG.warning to match log statement in exception.

Reply:
Combined the two statements.